sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Models/Identity.cs

```diff
@@ -1,44 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-// <auto-generated/>
-
-#nullable disable
-
-using System.Collections.Generic;
-using Azure.Core;
-
-namespace Azure.ResourceManager.Core
-{
-    /// <summary> Identity for the resource. </summary>
-    public partial class Identity
```

Comment:
Where did we move this class?

Reply:
This was a duplicate with ResourceIdentity so its not needed